core/src/main/java/org/sonarsource/sonarlint/core/rules/ActiveRuleDetailsAdapter.java

```diff
@@ -143,12 +139,25 @@ private static Collection<ActiveRuleDescriptionTabDto> transformSectionsButIntro
               DEFAULT_CONTEXT_KEY, DEFAULT_CONTEXT_DISPLAY_NAME));
           content = Either.forRight(new ActiveRuleContextualSectionWithDefaultContextKeyDto(DEFAULT_CONTEXT_KEY, contextualSectionContents));
         }
-        tabbedSections.add(new ActiveRuleDescriptionTabDto(title, content));
+        tabbedSections.add(new ActiveRuleDescriptionTabDto(getTabTitle(ruleDetails, sectionKey), content));
       }
     });
     return tabbedSections;
   }
 
+  private static String getTabTitle(ActiveRuleDetails ruleDetails, String sectionKey) {
+    if (ROOT_CAUSE_SECTION_KEY.equals(sectionKey)) {
+      return RuleType.SECURITY_HOTSPOT.equals(ruleDetails.getType()) ? "What's the risk?" : "Why is this an issue?";
+    }
+    if (ASSESS_THE_PROBLEM_SECTION_KEY.equals(sectionKey)) {
+      return "Assess the risk";
+    }
+    if (HOW_TO_FIX_SECTION_KEY.equals(sectionKey)) {
+      return "How can I fix it?";
+    }
+    return "More Info";
```

Comment:
Shouldn't we explicitely test if sectionKey == RESOURCES_SECTION_KEY?
In case of unknown section key, we could log a warning and simply return the key.

Reply:
At this point it's not possible, the sectionKey parameter has been validated earlier

Reply:
ok, I see